.circleci/config.yml

```diff
@@ -509,6 +510,10 @@ jobs:
       - run:
           working_directory: ui-v2
           command: node_modules/ember-cli/bin/ember exam --split=$CIRCLE_NODE_TOTAL --partition=`expr $CIRCLE_NODE_INDEX + 1` --path dist --silent -r xunit
```

Comment:
I'm not familiar with `ember exam`, is there additional config that would be needed to output a code coverage report from here similar to the [`test:coverage`](https://github.com/hashicorp/consul/blob/master/ui-v2/package.json#L26) npm script @johncowen? (the CodeCov upload doesn't seem to be uploading any frontend test coverage data currently)